### DISCUSSION TITLE
How can I sign in to two separate servers in the same session?

### DISCUSSION THREAD

**Discussion by JameT55** [codefori/vscode-ibmi#1416] [IBM i] [answered]
When I bring up VS Code, and select the IBMi icon, I see two servers, one for production and one for test.  I would like to open them both so that I can do a source file comparison.  The problem is that when I open up one of the servers, the "SERVERS" screen goes away, so I can't open up the other server.   

Does anyone have any ideas on how to solve this one?

![image](https://github.com/halcyon-tech/vscode-ibmi/assets/29083265/ba1ae485-68e1-46ba-b81f-7027d2f91a46)

**Comment by chrjorgensen** [comment]
You start another instance of VS Code and connect to the other system here.

Don't know if source compare works across connections, though... @worksofliam ?

**Reply by JameT55** [reply]
I have RDi and have been working on comparing that with VSCode for my team.  In RDi, I can bring up the production source and compare it to test source.  It works great there, maybe that's why it costs money!

**Comment by Wright4i** [comment, marked as answer]
Like @chrjorgensen said open a new window (Ctrl+Shift+N/Cmd+Shift+N) and open your second connection there.

Comparing source using the `Compare with..` commands won't work across windows however. Best work around I've found (and I haven't looked long) was to copy my source in one window into the clipboard and use `Compare Active File with Clipboard` on the other window.

**Reply by JameT55** [reply]
That is just crazy enough to work!!!   Thanks!
